### ISSUE TITLE
Can not download query engine for openssl 3.2

### ISSUE THREAD

**Issue by maxmousse** [Brendonovich/prisma-client-rust#425]
Hello everyone,

While trying to run the prismal-cli, I enconter the next error: 

```
Downloading https://binaries.prisma.sh/all_commits/d6e67a83f971b175a593ccc12e15c4a757f93ffe/debian-openssl-3.2.x/query-engine.gz to /home/maxim/.cache/prisma/binaries/cli/4.8.0/d6e67a83f971b175a593ccc12e15c4a757f93ffe/prisma-query-engine-debian-openssl-3.2.x
thread 'main' panicked at /home/maxim/.cargo/git/checkouts/prisma-client-rust-fa967aa5ad0ec391/948642d/cli/src/binaries/mod.rs:145:9:
received code 404 Not Found from https://binaries.prisma.sh/all_commits/d6e67a83f971b175a593ccc12e15c4a757f93ffe/debian-openssl-3.2.x/query-engine.gz
```

I am on an arched base distribution and moved from openssl 3.1 to 3.2 on the 17th of december.

From what I understand, it seems to be the same problem than in issue #360 no ?

Once a solution is found, I'd be glad to submit a pull request if it can help =)

**Comment by Brendonovich** [comment]
This is fixed on `0.6.x`, just need to cut a release

**Comment by maxmousse** [comment]
Hello @Brendonovich, 

Thank you for your answer. Awesome, so I'll just rollback to openssl 3.1 while waiting for the next release =)

**Comment by Brendonovich** [comment]
Fixed in ad328c5
